lib/vagrant-service-manager/command.rb

```diff
@@ -5,6 +5,7 @@ module Vagrant
   module ServiceManager
     DOCKER_PATH = '/home/vagrant/.docker'
     SUPPORTED_SERVICES = ['docker', 'openshift', 'kubernetes']
+    SCCLI_SERVICES = ["openshift", "kubernetes", "k8s"]
```

Comment:
If we think Kubernetes and k8s both are required then we should fix sccli to include Kubernetes and then use it here. That way we would be consistent across the board.